tasks/91_make-a-person/make-a-person.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @description
```

Comment:
Work with private fields - smth like a   
<code>constructor(fullName) {
    [this._firstName, this._lastName] = fullName.split(' ');
  }
</code>